packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -179,15 +179,23 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     ]
   );
 
+  const messageContentRef = useRef<HTMLDivElement | null>(null);
+
   const chatMessage = (
     <>
-      <div ref={messageRef}>
+      <div
+        ref={messageRef}
+        tabIndex={0}
+        onFocus={() => {
```

Comment:
Yes, If I don't have a good 

> After testing on my iPhone SE with Miguel, everything looks good. Only additional thing to add is announcing the message author with the message and possibly the timestamp as indicated in the ado bug.

Timestamp might be complicated because the timestamp needs to be localizable like "Greg said Hi on Thursday 11:43am". Will look into some options with [toLocaleDateTimeString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString)